src/main/java/org/tron/common/runtime/vm/program/Storage.java

```diff
@@ -3,15 +3,19 @@
 import static java.lang.System.arraycopy;
 import java.util.HashMap;
 import java.util.Map;
+import lombok.Getter;
 import org.tron.common.crypto.Hash;
 import org.tron.common.runtime.vm.DataWord;
 import org.tron.core.capsule.StorageRowCapsule;
 import org.tron.core.db.StorageRowStore;
 
 public class Storage {
 
+  @Getter
   private byte[] addrHash;  // contract address
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this trailing comment on the previous empty line. ([squid:TrailingCommentCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ATrailingCommentCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6qGSgN4uCclo1de&open=AWbJU6qGSgN4uCclo1de)

[ISSUE_KEY]: AWbJU6qGSgN4uCclo1de